contracts/near/eth-client/src/tests.rs

```diff
@@ -262,8 +262,47 @@ fn add_dags_merkle_roots() {
     assert_eq!(dmr.dag_merkle_roots[0], contract.dag_merkle_root(0));
     assert_eq!(dmr.dag_merkle_roots[10], contract.dag_merkle_root(10));
     assert_eq!(dmr.dag_merkle_roots[511], contract.dag_merkle_root(511));
+    assert_eq!(dmr.dag_merkle_roots[699], contract.dag_merkle_root(699));
 
-    let result = catch_unwind_silent(|| contract.dag_merkle_root(512));
+    let result = catch_unwind_silent(|| contract.dag_merkle_root(700));
+    assert!(result.is_err());
```

Comment:
Add a comment explaining this panic.

Reply:
916926ee217d5b7c0291973b306b361aedf20c29